src/evm/contracts/ILightClient.sol/INonMembershipVerifier.ts

```
@@ -0,0 +1,135 @@
+
+  listeners<TCEvent extends TypedContractEvent>(
+    event: TCEvent
+  ): Promise<Array<TypedListener<TCEvent>>>;
+  listeners(eventName?: string): Promise<Array<Listener>>;
+  removeAllListeners<TCEvent extends TypedContractEvent>(
+    event?: TCEvent
+  ): Promise<this>;
+
+  verifyNonMembership: TypedContractMethod<
+    [proof: Ics23ProofStruct, key: BytesLike],
+    [void],
```

Biome (1.9.4)

[error] void is confusing outside a return type or a type parameter.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)